frontend-project/src/components/Table/ChannelName.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Spin } from 'antd';
+
+import { fetchChannel } from '@/services/channel';
+
+export const ChannelName: FC<{ id: number }> = props => {
+  const [name, setName] = useState('');
```

Comment:
tutaj to samo pytanie odnośnie trzymania tego w storze reduxowym (przy czym możemy pewnie ten temat zaadresować, na żywo za tydzień na spotkaniu).

Reply:
zajmę się przeniesieniem kanału do stora reduxowego (również instytucji i sprawy)